corehq/ex-submodules/casexml/apps/phone/tests/performance_tests.py

```diff
@@ -21,6 +21,11 @@
 REFERRAL_TYPE = 'referral'
 
 
+def setupModule():
```

Comment:
Good to know. This is already merged so not going to change it but feel free to go with `@skip` if you want @benrudolph